tests/unit/test_campaign_model.py

```
@@ -0,0 +1,222 @@
+"""Unit tests for CampaignModel, CampaignEventModel, TriggerModel ORM to_dict."""
+
+from __future__ import annotations
+
+import pytest
+from datetime import datetime
+
+from tests.unit.conftest import MockState, make_mock_session, make_campaign_handler
```

Pylint (4.0.5)

[error] No name 'make_campaign_handler' in module 'tests.unit.conftest'

(E0611)